etc/README.md

```diff
@@ -0,0 +1,46 @@
+# Installation instructions
+
+## bash
+
+Open your `.bashrc` file and add:
+
+> If you want to set up Git to automatically have Bash shell completion for all users, copy the `hub.bash_completion` script to the `/opt/local/etc/bash_completion.d` directory on Mac systems or to the `/etc/bash_completion.d/` directory on Linux systems. ([Source](https://git-scm.com/book/en/v1/Git-Basics-Tips-and-Tricks#Auto-Completion))
```

Comment:
Blockquote is usually reserved for quoting what somebody said, for instance if I reply to you and quote a part of what you said, just like in email:

> Do you think it's better to just remove it?